src/main/java/com/dao/Database.java

```diff
@@ -0,0 +1,39 @@
+package com.dao;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.model.movies.Movies;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+@Component
+public class Database {
+
+    private Movies moviesList;
```

Comment:
At present, we're cacheing the `Movies` object here but the shape of this data resembles 
```
{
   "movies": [
      {
          ...
      },
      ...
   ]
}
```
Would it make more sense to cache a Collection of `Movie` objects on the class instead, either as an array or a list?